conf/sample/sidecar-injector/ServiceAccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I wonder if we may want this sidecar config simply in a directory called `conf/sidecar-injector`? We are quickly outgrowing stuffing all sample config in the `conf/sample` directory. :)